### PR TITLE
MM-12319 - Updating x icon css in search bar

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
MM-12319 - Updating x icon css in search bar

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-12319

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Ran `make check-style` to check for style errors (required for all pull requests)
- [ ] Ran `make test` to ensure unit and component tests passed
- [ ] Added or updated unit tests (required for all new features)
- [ ] Needs to be implemented in mobile (link to PR or User Story)
- [ ] Has server changes (please link)
- [ ] Has redux changes (please link)
- [x] Has UI changes
- [ ] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, posting, etc.)

### CODE REVIEW

**Comment by lindy65** [comment]
Thanks @asaadmahmood 

After testing on the spinmint, and looking at the expected from the Jira ticket:

> Add a "x" on the right-hand side of the search input bar that will allow users to remove all search input from the search bar after clicking it. 
- Tested and correct

> Hovering over the "x" should produce a tool tip "Clear search query". 
- I do not see a tooltip on hover

> Once the user has pressed the "x", the Search Options modal should be shown, but any results from a previous search would not be removed.
- Tested and working as expected (tested "jump" to message as well as "reply" to message from search results as well and this is working as expected)

> The "x" should not be present until there is at least one character entered in the search term box.
- Tested and working as expected

I see on the ticket that implementation is for RN apps as well.  
- There is already an "X" to the right of the search term on the RN apps that clears the search term as well as any search results and redirects you back to the "Search filters screen" (i.e. "from:[username]", "in:[channelname]", etc.).
- Should the Jira ticket description be updated to exclude the RN apps as it seems this feature is already working there (and there is no "on hover" effect or tooltip in RN)?

**Comment by asaadmahmood** [comment]
@lindy65 Added the tooltip, and I think in the ticket we can just mention that it is already implemented on RN.

**Comment by lindy65** [comment]
Thanks @asaadmahmood 

I'll just wait for the spinmint to rebuild and will retest.  I'll also update the Jira ticket to include RN comment.

**Comment by lindy65** [comment]
Thanks @asaadmahmood 

I have retested as per ticket and everything is working as expected as well as the tooltip on hover being visible.

File: components/search_bar/search_bar.jsx [hunk 1]
```diff
@@ -223,6 +224,15 @@ export default class SearchBar extends React.Component {
             searchFormClass += ' focused';
         }
 
+        const searchClearTooltip = (
+            <Tooltip id='searchClearTooltip'>
+                <FormattedMessage
+                    id='search_bar.clear'
+                    defaultMessage='Members'
```
Comment (saturninoabril) [inline components/search_bar/search_bar.jsx]: Should change to "Clear search query"

**Comment by cpanato** [comment]
`Setup Test Server` label detected. Spinmint test server created if build succeeds (checks pass and no conflicts with base branch).

**Comment by cpanato** [comment]
Spintmint test server creation failed. See the logs for more information.

**Comment by asaadmahmood** [comment]
@cpanato Should work now.

**Pull request merged**